### ISSUE TITLE
Error: No output file

### ISSUE THREAD

**Issue by traini82** [milaboratory/mixcr#352]
Trying to align single end rep-seq reads using following invocation and can't get past the 'No output file' error. I assume its looking for the paired read but this is not '-p RNA-seq' sequence but enriched amplicon-seq reads. Please advise.

mixcr align -s mmu -c TCR -f -r TCRm5-1_R1.fastq.gz TCRm5-1_R1_alignments.vdjca

**Comment by PoslavskySV** [comment]
You use ``-r`` option not correctly. Try

```
mixcr align -s mmu -c TCR -f -r report.txt TCRm5-1_R1.fastq.gz TCRm5-1_R1_alignments.vdjca
```

**Comment by dbolotin** [comment]
I also suggest running `align` without `-c TCR`.

**Comment by traini82** [comment]
Thanks, 

I mistakenly thought ‘-r’ without {file} would default to screen output.
In regards to the ‘write all’ option I’ve used both ‘-v’ and –write-all’ and still see the following output warning;

“WARNING: 6 functional genes were excluded, re-run with -v option to see the list of excluded genes and exclusion reason.”

Am I missing a qualifier?

**Comment by traini82** [comment]
Thanks Dmitry,

I mistakenly thought ‘-r’ without {file} would default to screen output.
In regards to the ‘write all’ option I’ve used both ‘-v’ and –write-all’ and still see the following output warning;

“WARNING: 6 functional genes were excluded, re-run with -v option to see the list of excluded genes and exclusion reason.”

Am I missing a qualifier?

Thanks,

Chris

**Comment by PoslavskySV** [comment]
Regarding the “WARNING: 6 functional genes were excluded, re-run with -v option to see the list of excluded genes and exclusion reason.” It was a bug which is now fixed (in version 2.1.9). The correct option to see the list of excluded genes is ``--verbose``.

**Comment by dbolotin** [comment]
MiXCR by default prints report to stdout after all main actions execution, `-r` is needed to also write this information to file.

Please use `--write-all` instead of `-v`, as @PoslavskySV mentioned, we just fixed all misleading warning messages and have deprecated `-v` option in 2.1.9.

I also, strongly suggest upgrading to 2.1.9, it, among other things, contains upgraded segment library (see changelog for the library [here](https://github.com/repseqio/library/releases/tag/v1.5)), an important change is that TRAV/TRDV genes were assigned with both chains, previously some TRAV/DV genes were assigned with either TRA or TRD chain, which in turn led in some cases to elimination of correct rearrangements as chimeric. In real data, pretty much any of TRAV/DV / TRAJ/DJ combinations happen from time to time, so we removed any limitations here.
